eidas-saml-parser/src/main/java/uk/gov/ida/notification/eidassaml/logging/EidasAuthnRequestAttributesLogger.java

```diff
@@ -18,7 +18,10 @@ public static void logAuthnRequestAttributes(AuthnRequest authnRequest) {
         } catch (Exception e) {
```

Comment:
Can remove this, it's redundant.

Reply:
Sure. It was already there but I'll take it out.